save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -264,7 +109,7 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
 
             if (isSuperAdmin || isOwner) {
                 buttonBuilder(label = "Delete", style = "danger", classes = "mr-2") {
-                    setDeleteProject(null)
+                    // setDeleteProject(null)
```

Comment:
Why so?

Reply:
Forgot to delete.
It is strange that the Diktat did not notice this.

Reply:
Sometimes diktat skips those commented code fragments